build fail looks real
```
======================================================================
ERROR: test_create_form_export (corehq.apps.export.tests.test_export_views.ExportViewTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/export/tests/test_export_views.py", line 48, in test_create_form_export
    {'export_tag': 'my_sweet_xmlns'}
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 470, in get
    **extra)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 286, in get
    return self.generic('GET', path, secure=secure, **r)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 358, in generic
    return self.request(**r)
  File "/vendor/lib/python2.7/site-packages/django/test/client.py", line 440, in request
    six.reraise(*exc_info)
  File "/vendor/lib/python2.7/site-packages/django/core/handlers/base.py", line 111, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/vendor/lib/python2.7/site-packages/django/views/generic/base.py", line 69, in view
    return self.dispatch(request, *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/style/decorators.py", line 303, in _wrapped
    return view_func(class_based_view, request, *args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/domain/decorators.py", line 101, in _inner
    return view_func(req, domain_name, *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/users/decorators.py", line 21, in _inner
    return view_func(request, domain, *args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/django/utils/decorators.py", line 25, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/mnt/commcare-hq-ro/corehq/apps/export/views.py", line 1476, in dispatch
    return super(BaseModifyNewCustomView, self).dispatch(request, *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/style/decorators.py", line 89, in _wrapped
    return view_func(class_based_view, request, *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/export/views.py", line 1436, in dispatch
    return super(BaseNewExportView, self).dispatch(request, *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/style/decorators.py", line 89, in _wrapped
    return view_func(class_based_view, request, *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/export/views.py", line 154, in dispatch
    return super(BaseExportView, self).dispatch(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/django/utils/decorators.py", line 29, in _wrapper
    return bound_func(*args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/domain/decorators.py", line 101, in _inner
    return view_func(req, domain_name, *args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/django/utils/decorators.py", line 25, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/mnt/commcare-hq-ro/corehq/apps/domain/views.py", line 215, in dispatch
    return super(LoginAndDomainMixin, self).dispatch(*args, **kwargs)
  File "/vendor/lib/python2.7/site-packages/django/views/generic/base.py", line 87, in dispatch
    return handler(request, *args, **kwargs)
  File "/mnt/commcare-hq-ro/corehq/apps/export/views.py", line 1491, in get
    xmlns,
  File "/mnt/commcare-hq-ro/corehq/apps/export/models/new.py", line 948, in generate_schema_from_builds
    app_build_ids.extend(cls._get_current_app_ids_for_domain(domain))
  File "/mnt/commcare-hq-ro/corehq/apps/export/models/new.py", line 1083, in _get_current_app_ids_for_domain
    raise BadExportConfiguration('Form exports should only use one app_id and this should not be called')
BadExportConfiguration: Form exports should only use one app_id and this should not be called
```

--- a/corehq/apps/export/models/new.py
+++ b/corehq/apps/export/models/new.py
@@ -959,7 +959,7 @@ class ExportDataSchema(Document):
                 identifier,
             )
 
-            current_schema.record_update(app.copy_of or app_id, app.version)
+            current_schema.record_update(app.copy_of or app._id, app.version)
 
         current_schema.domain = domain
         current_schema.app_id = app_id

--- a/corehq/apps/export/tests/test_export_views.py
+++ b/corehq/apps/export/tests/test_export_views.py
@@ -45,7 +45,7 @@ class ExportViewTest(TestCase):
     def test_create_form_export(self):
         resp = self.client.get(
             reverse(CreateNewCustomFormExportView.urlname, args=[self.domain.name]),
-            {'export_tag': 'my_sweet_xmlns'}
+            {'export_tag': 'my_sweet_xmlns', 'app_id': 'r2d2'}
         )
         self.assertEqual(resp.status_code, 200)
 

